src/Spreadsheet/PHPExcel/Font.php

```diff
@@ -0,0 +1,148 @@
+<?php
+declare(strict_types=1);
+
+namespace HelloFresh\Ausraster\Spreadsheet\PHPExcel;
+
+use PHPExcel_Font;
+use HelloFresh\Ausraster\FontInterface;
+
+class Font implements FontInterface
+{
+    /**
+     * Typeface name.
+     * @var string
+     */
+    private $name = 'Calibri';
+
+    /**
+     * Font size.
+     * @var int
+     */
+    private $size = 12;
+
+    /**
+     * Font bold status.
+     * @var bool
+     */
+    private $bold = false;
+
+    /**
+     * Font italic status.
+     * @var bool
+     */
+    private $italic = false;
+
+    /**
+     * Font underline status.
+     * @var bool
+     */
+    private $underline = false;
+
+    /**
+     * Font color.
+     * @var string
+     */
+    private $color = '000000';
+
```

Comment:
Wouldn't be good to have a separate static class for all of the colors?, for example

```php
RGB::Black
```